scripts/tests/test-ci-all.sh

```diff
@@ -253,7 +253,7 @@ fi
 
 parallel_args+=(--timeout "${FM_TEST_CI_ALL_TIMEOUT:-600}")
 
-parallel_args+=(--load "${FM_TEST_CI_ALL_MAX_LOAD:-1000}")
+parallel_args+=(--load "${FM_TEST_CI_ALL_MAX_LOAD:-$(nproc) / 4 + 1}")
```

Comment:
Ahh we need to surround with `$(())` for bash arithmetic 😄 

It tried to run it like this

```
## Starting all tests in parallel...
parallel --jobs 9 --timeout 600 --load 32 / 4 + 1 --delay 3 --halt-on-error 1 --joblog /tmp/nix-shell.43NPCg/Tys1T/joblog --noswap --memfree 2G
```

Reply:
I've also verified locally a different approach that uses `%`

```
parallel_args+=(--load "${FM_TEST_CI_ALL_MAX_LOAD:-$((($(nproc) / 4 + 1) * 100))%}")
```